pages/linux/tomb.md

```diff
@@ -10,6 +10,10 @@
 
 `tomb forge {{encrypted_directory.tomb.key}}`
 
+- Forcefully create a new key if tomb isn't letting key forging due to swap:
```

Comment:
Perhaps this will read more clearly?
```suggestion
- Forcefully create a new key, even if the tomb isn't allowing key forging (due to swap):
```
(also, what does "swap" mean in this case?)

Reply:
You suggestion is better. Thanks. 
Swap, means the "swapfile" or "swap partition" {i.e., swapiness) . It is a problem faced by newbies, where they aren't able to do a lot of actions. The recommended way is to turn of swap (`swapoff`), but that's not always ideal (& not always required if the device is a PC, with no threat), in which case one can give the `-f` option.